azure-quantum/requirements-qsharp.txt

```diff
@@ -1 +1 @@
-qsharp>=0.28.263081,<=1.0
\ No newline at end of file
+qsharp-lang>=1.0.0.dev0,<2.0
```

Comment:
this dependency on [qsharp-lang](https://pypi.org/project/qsharp-lang/) package specifically is WIP until releases `qsharp` vs `qsharp-lang` are figured out.

Reply:
Updated the requirements to use the latest `qsharp==1.0.30.dev0`